CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## NEXT
 ### Fixes
 * Service init order  [#131](https://github.com/TGFTech/angular-cesium/issues/131)
```

Comment:
add the service to services.ts and export it

Reply:
done